tools/astarte_dev_tool/lib/commands/auth/keys.ex

```diff
@@ -0,0 +1,26 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteDevTool.Commands.Auth.Keys do
+  @moduledoc false
+  alias AstarteDevTool.Utilities.Auth, as: AuthUtilities
```

Comment:
I will not define an alias like this 
```
alias AstarteDevTool.Utilities.Auth, as: AuthUtilities
``` 
then below, alias the same module just as 
```
alias AstarteDevTool.Utilities.Auth
```

Since the only occurrence of Auth in this file is within the Utilities group, we can just alias as Auth
```suggestion
  alias AstarteDevTool.Utilities.Auth 
```